docs/source/logging.rst

```diff
@@ -1,15 +1,14 @@
-neomaril\_codex Logging module
-==============================
+Logging module
+==============
 
-Module contents
----------------
 
+Neomaril has a log parser that clean logs and tries to find useful information. This is the module to access this functionalities.
+Since there are a lot of logger types is hard to create the perfect parser, so using this one helps your model script being parsed by Neomaril and sending cleaner messages.
 
-neomaril\_codex.logging
------------------------
 
-.. automodule:: neomaril_codex.logging
-   :members:
-   :undoc-members:
-   :show-inheritance:
+neomaril\_codex.logging.Logger
+------------------------------
 
+.. autoclass:: neomaril_codex.logging.Logger
+   :members: debug, error, warning, callback
+   :private-members: +__log
```

Comment:
I think we can remove all private-members, the users wont be able to call those methods anyway.

Reply:
Maybe we can turn the base.py module public, but the private methods i think we can hide.